scripts/drtprod

```diff
@@ -456,6 +456,172 @@ WantedBy=multi-user.target
+        $0 sql drt-ldr1:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr2' AS '${ldr2}'"
+        $0 sql drt-ldr2:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr1' AS '${ldr1}'"
+        roachprod sql drt-ldr1:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE ycsb.usertable ON 'external://drt-ldr2' INTO TABLE ycsb.public.usertable;"
+        roachprod sql drt-ldr2:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE ycsb.usertable ON 'external://drt-ldr1' INTO TABLE ycsb.public.usertable;"
+
+        echo
+        echo "Completed setup of ldr1 and ldr2, with logical data replication between them on table ycsb.usertable."
+        echo
+        echo "Run 'drtprod create workload-ldr' to create the workload node and start the workload."
+        echo
+        ;;
+      "workload-ldr")
+        set -x
+        roachprod create workload-ldr \
+          --clouds="gce" \
+          --gce-zones="us-east1-c" \
+          --nodes="1" \
+          --gce-machine-type="n2-standard-16" \
+          --os-volume-size 100 \
+          --username workload \
+          --lifetime 8760h \
+        # setup dns
+        $0 dns workload-ldr create || $0 dns workload-ldr
+
+        # push cockroach to the worker to run its built-in workloads
+        roachprod stage workload-ldr:1 cockroach
+        roachprod stage workload-ldr:1 workload
+
+        # Push certs for both clusters to the worker.
+        roachprod get drt-ldr1:1 certs certs.ldr1
+        roachprod put workload-ldr:1 certs.ldr1
+        roachprod get drt-ldr2:1 certs certs.ldr2
+        roachprod put workload-ldr:1 certs.ldr2
+        roachprod ssh workload-ldr:1 -- "chmod 0600 certs.ldr1/* certs.ldr2/*"
+        rm -rf certs.ldr1 certs.ldr2
+
+        # Setup pgurls for worklooads to use.
+        roachprod ssh workload-ldr:1 -- "
+        echo \"$(roachprod pgurl drt-ldr1 --secure | sed 's/=certs/=certs.ldr1/g' | sed "s/'//g" )\" > pgurls.ldr1.txt;
+        echo \"$(roachprod pgurl drt-ldr2 --secure | sed 's/=certs/=certs.ldr2/g' | sed "s/'//g" )\" > pgurls.ldr2.txt;
+        ln -sf pgurls.ldr1.txt pgurls.txt"
+
+        # Setup the ycsb workload runner as a script invoked by systemd.
+        # The runner cycles through all lettered ycsb workloads in a
+        # round-robin fashion, plus an insert-only workload.
+        roachprod ssh workload-ldr:1 -- 'cat - > ycsb_run_ldr1.sh << EOF
+#!/usr/bin/env bash
+
+workloads=(A B C D E F)
+
+while true; do
+  for workload in "\${workloads[@]}"; do
+    exec ./cockroach workload run ycsb \\
+      --concurrency=32
+      --workload=\$workload \\
+      --duration=2h \\
+      --ramp=5m \\
+      \$(cat /home/ubuntu/pgurls.ldr1.txt)
+  done
+
+  exec ./cockroach workload run ycsb \\
+    --concurrency=32
+    --workload='custom' --insert-freq=1.0 \\
+    \$(cat /home/ubuntu/pgurls.ldr1.txt)
+done
+EOF'
+        roachprod ssh workload-ldr:1 -- 'cat - > ycsb_run_ldr2.sh << EOF
+#!/usr/bin/env bash
+
+workloads=(A B C D E F)
```

Comment:
Nit: duplicate definition of workloads

Reply:
Necessary because it's a separate script